CHANGES.md

```diff
@@ -2,6 +2,7 @@ Airbase 128
 
 * Plugin updates:
   - JaCoCo 0.8.8 (from 0.8.7)
+* Fix configuration of Surefire when Jacoco is disabled via property in POM
```

Comment:
I think we're fixing IntelliJ's behavior

Reply:
Sure, but who knows what other tools are confused by this as well...

Reply:
well, we know that Surefire maven plugin is not confused, so we shouldn't say we're fixing "configuration for Surefire".

Reply:
How about something like this?
```suggestion
* Ensure ``argLine`` is always defined, with or without Jacoco being enabled
```

Reply:
I'll work for me :) The details are in the commit message (another Leo Tolstoy badge guaranteed)